src/modules/dashboard/components/ManageFundsDialog/ManageFundsDialog.tsx

```diff
@@ -107,6 +114,14 @@ const ManageFundsDialog = ({
       title: MSG.mintTokensTitle,
       description: MSG.mintTokensDescription,
       icon: 'emoji-seed-sprout',
+      permissionRequired: !canMintNativeToken,
+      permissionInfoText: MSG.permissionsListText,
+      permissionInfoTextValues: {
+        permissionsList: (
+          <FormattedMessage {...MSG.mintTokensPermissionsList} />
+        ),
+      },
+      onClick: () => callStep(nextStepMintTokens),
```

Comment:
Currently we have a bug here that breaks the `canMintNativeToken` logic

![Screenshot from 2021-01-06 19-03-00](https://user-images.githubusercontent.com/1193222/103800451-2057cf80-5055-11eb-82f1-2cb261888322.png)

It's to do with the Apollo cache and our `ethereal` wallet.

Please open an issue to track this, as it needs to be fixed prior to launch